sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -62,14 +62,14 @@ class RunningCountStatefulProcessor extends StatefulProcessor[String, String, (S
 
 class RunningCountMostRecentStatefulProcessor
   extends StatefulProcessor[String, (String, String), (String, String, String)]
-    with Logging {
+  with Logging {
   @transient private var _countState: ValueState[Long] = _
   @transient private var _mostRecent: ValueState[String] = _
   @transient var _processorHandle: StatefulProcessorHandle = _
 
   override def init(
-       handle: StatefulProcessorHandle,
-       outputMode: OutputMode) : Unit = {
+    handle: StatefulProcessorHandle,
```

Comment:
nit: 4 spaces for method params. https://github.com/databricks/scala-style-guide?tab=readme-ov-file#indent